app/routes/route_utilities.py

```diff
@@ -46,7 +46,11 @@ def send_slack_message_to_channel(message: str, channel: str):
     if not response.ok:
         print(f"Error sending message to Slack: {response.text}")
 
+def build_task_completion_message(task):
```

Comment:
👍 Great. In a fuller application, we might have a lot more functionality dedicated to the Slack integration, leading to additional classes and modules to organize them all together.